pkgs/servers/lenpaste/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitea, git }:
+
+buildGoModule rec {
+  pname = "lenpaste";
+  version = "1.3";
+
+  src = fetchFromGitea {
+    domain = "git.lcomrade.su";
+    owner = "root";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-d+FjfEbInlxUllWIoVLwQRdRWjxBLTpNHYn+oYU3fBc=";
+  };
+
+  buildInputs = [ git ];
+
+  ldflags = "-w -s -X main.Version=${version}";
+
+  fixupPhase = ''
+    mv $out/bin/lenpaste-unwrapped $out/bin/lenpaste
+  '';
+
+  vendorSha256 = "sha256-PL0dysBn1+1BpZWFW/EUFJtqkabt+XN00YkAz8Yf2LQ=";
```

Comment:
Consider moving `vendorSha256` (or the newer `vendorHash`) up a few lines between `src` and `ldflags`.

Both hashes in `src.sha256` (or the newer `src.hash`) and `vendorHash` need to be updated virtually every single version update.
Keeping them a bit closer together (line-wise) is usually preferred :)